Assets/Scripts/Entity/Player/PlayerController.cs

```diff
@@ -17,38 +19,37 @@ public class PlayerController : Movable
 
         [Tooltip("The clickable layers. Defines where the player can click")] [SerializeField]
         private LayerMask clickableLayers;
-        
+
         [Tooltip("Defines where the player can move")] [SerializeField]
         private LayerMask groundLayer;
 
 
-        [Tooltip("An effect that will be displayed whenever the player clicks to move")]
-        [SerializeField]
+        [Tooltip("An effect that will be displayed whenever the player clicks to move")] [SerializeField]
```

Comment:
haha, your work will be destroyed as soon as someone uses auto format :[

Reply:
This was auto-formatting, so no worries:D